.github/workflows/e2e.yml

```diff
@@ -9,7 +9,8 @@ on:
       - main
   schedule:
     - cron: '0 0 * * *'
-    
+  workflow_dispatch:
+
```

Comment:
Is the colon at the end compulsory? Can we remove it?

Reply:
yes its compulsory